ci/build_wheel_libcudf.sh

```diff
@@ -5,11 +5,15 @@ set -euo pipefail
 
 package_dir="python/libcudf"
 
+export SKBUILD_CMAKE_ARGS="-DUSE_CUDA_NVCOMP_WHEEL=ON"
```

Comment:
Should this be `USE_NVCOMP_WHEEL` (no CUDA)?

Reply:
I will update the name once this CI run is complete.

Reply:
Should it be `USE_NVCOMP_RUNTIME_WHEEL` to specify that it's only runtime? I'm OK with not, just a suggestion.

Reply:
Updated